ci-operator/step-registry/cucushift/installer/rehearse/vsphere/ipi/proxy/rhel8/provision/cucushift-installer-rehearse-vsphere-ipi-proxy-rhel8-provision-chain.yaml

```diff
@@ -0,0 +1,31 @@
+chain:
+  as: cucushift-installer-rehearse-vsphere-ipi-proxy-rhel8-provision
+  steps:
+  - chain: ipi-conf-vsphere
+  - ref: ipi-conf-vsphere-proxy
+  - chain: ipi-install-vsphere
+  - ref: ipi-conf-vsphere-bastionhost
+  - ref: workers-rhel-vsphere-provision
+  - ref: workers-rhel-pre-hook-sshtunnel
+  - chain: workers-rhel
+  env:
+  - name: RHEL_WORKER_COUNT
+    default: 2
+    documentation: The number of rhel worker to be provisioned.
+  - name: RHEL_IMAGE
+    default: "qe-rhel-85-release"
+    documentation: |-
+      The name of rhel vm/template existing under folder /<Datacenter>/vm/Templates on vSphere, used to clone RHEL instances
+  - name: PLATFORM_VERSION
+    default: 8.5
+    documentation: the version of rhel, valid value is 7.x and 8.x.
+  - name: RHEL_USER
+    default: "test"
+    documentation: |-
+      The user to login the system of RHEL instance
+  - name: REMOVE_RHCOS_WORKER
+    default: "no"
```

Comment:
To not change the original logic, I guess maybe making its fault value to be "yes" maybe better.